roles/reset/tasks/main.yml

```diff
@@ -264,6 +264,8 @@
   with_items:
     - "{{ kube_config_dir }}"
     - /var/lib/kubelet
+    - /var/lib/containerd
+    - /var/lib/etcd
```

Comment:
`"{{ etcd_data_dir }}"` is specified already at the original line 269.

Reply:
Hmm, indeed, then I wonder what were the specific conditions in https://github.com/kubernetes-sigs/kubespray/issues/8449 where it did not get removed.

/cc @wirwolf